packages/core/src/render3/after_render_hooks.ts

```diff
@@ -188,6 +188,20 @@ export function internalAfterNextRender(
   afterRenderEventManager.internalCallbacks.push(callback);
 }
 
+/**
+ * Registers the given callback as a DomRef initializer, ensuring
+ * that it will be run prior to any user after*Render callbacks.
+ */
+export function registerDomRefInitializer<T>(callback: VoidFunction) {
+  const injector = inject(Injector);
+
+  // DomRef initializers are only registered on the browser.
+  if (!isPlatformBrowser(injector)) return;
```

Comment:
The helper function is able to inject the Injector when run in an injection context. 
```suggestion
  if (!isPlatformBrowser()) return;
```